scripts/end_to_end_test/end_to_end_test.yaml

```diff
@@ -133,7 +133,7 @@ steps:
 
 ---
 test_description: >
-  Test synergetic mining
+  DISABLED - Test synergetic mining
```

Comment:
Is this meant to be disabled at this time?

Reply:
It's disabled on master